di-ipv-ticf-stub/lambdas/src/services/ticfService.ts

```diff
@@ -24,7 +35,11 @@ export async function processGetVCRequest(
     nbf: timestamp,
     iat: timestamp,
     vc: {
-      evidence: [getEvidenceItem(timeoutVc, includeCi)],
+      evidence: [
+        !ticfEvidenceItem
+          ? getEvidenceItem(timeoutVc, includeCi)
+          : ticfEvidenceItem,
```

Comment:
Do you think the `timeoutVc` and `includeCi` parameters are still useful, now that we've added the management API?